PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+Resolves #
```

Comment:
Basically that would be the blank part of `The purpose of this pull request is to ________________________________________.`